wc.dd

```diff
@@ -31,7 +31,7 @@ int main(string[] args)
         foreach(line; file.byLine(KeepTerminator.yes))
         {
             lCharCount += line.length;
-            foreach(char[] word; splitter(line))
+            foreach(string word; splitter(line))
```

Comment:
`byLine` uses a mutable buffer, so this shouldn't compile better just remove the type completely.